e2e/kitchen_input_radio_checkbox/test_input_radio_checkboxgroup.py

```diff
@@ -77,3 +77,78 @@ def assert_selected(
     check2.set(["orange", "teal"])
 
     assert_selected("b", "f", ["red", "blue"], ["orange", "teal"])
+
+
+def test_locator_debugging(page: Page, local_app: ShinyAppProc) -> None:
+    page.goto(local_app.url)
+    # Fail quickly
+    timeout = 100
+
+    # Non-existent div
+    try:
+        not_exist = InputRadioButtons(page, "does-not-exist")
+        not_exist.expect_choices(["a", "b", "c"], timeout=timeout)
+    except AssertionError as e:
+        assert "expected to have count '1'" in str(e)
+        assert "Actual value: 0" in str(e)
+
+    check1 = InputCheckboxGroup(page, "check1")
+
+    # Make sure it works
+    check1.expect_choices(["red", "green", "blue"])
```

Comment:
Make sure it can accept tuples in addition to lists.

Reply:
`typing.Tuple` needs a known size. While `typing.Sequence` works for tuples, lists, and range, Playwright only accepts `typing.List[]` objects. So we would be converting them to Lists in all of these places. 

Not pursuing. Resolving comment.